cmd/get/task_util.go

```diff
@@ -42,18 +43,7 @@ func FetchTaskForName(ctx context.Context, name string, project string, domain s
 }
 
 func FetchAllVerOfTask(ctx context.Context, name string, project string, domain string, cmdCtx cmdCore.CommandContext) ([]*admin.Task, error) {
-	tList, err := cmdCtx.AdminClient().ListTasks(ctx, &admin.ResourceListRequest{
-		Id: &admin.NamedEntityIdentifier{
-			Project: project,
-			Domain:  domain,
-			Name:    name,
-		},
-		SortBy: &admin.Sort{
-			Key:       "created_at",
-			Direction: admin.Sort_DESCENDING,
-		},
-		Limit: 100,
-	})
+	tList, err := cmdCtx.AdminClient().ListTasks(ctx, buildResourceListRequestWithName(config.GetConfig(), name))
```

Comment:
The other parameters of project , domain are not getting used in this function.
Please check other places aswell.

Reply:
Also as this is util function, add the config as function parameter and have higher level classes inject it in